src/java/main/com/epam/hud/dao/implement/ShopDAOImpl.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.hud.dao.implement;
+
+import com.epam.hud.dao.db.ConnectionPool;
+import com.epam.hud.dao.entity.AnimeShop;
+import com.epam.hud.dao.interfaces.ShopDAO;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+
+public class ShopDAOImpl implements ShopDAO {
+
+    private final String READ_SHOP = "SELECT * from anime.shops WHERE id=?;";
+    private final String CREATE_SHOP = "INSERT into anime.shops (name,address) VALUES (?,?);";
+    private final String UPDATE_SHOP = "UPDATE anime.shops SET name=?, address=? where id=?";
+    private final String DELETE_SHOP = "DELETE from anime.shops where id=?";
+    private final String READ_ALL_SHOPS = "SELECT * from anime.shops";
+
+    @Override
+    public void createShop(AnimeShop animeShop) throws SQLException {
+        Connection connection = ConnectionPool.getInstance().getConnection();
+        PreparedStatement statement = connection.prepareStatement(CREATE_SHOP);
+        statement.setString(1, animeShop.getShopName());
+        statement.setString(2, animeShop.getShopAddress());
+        statement.executeUpdate();
+        connection.close();
+    }
+
+
+    public AnimeShop readShop(int id) throws SQLException {
+        AnimeShop shop = new AnimeShop(); // Making object of class AnomeShop to set necessary data there.
+        Connection connection = ConnectionPool.getInstance().getConnection();
+        PreparedStatement statement = connection.prepareStatement(READ_SHOP); //
```

Comment:
add try-catch block and use finally block for closing connection
also need to add preparedStatement close()